extensions/cep/modules/distribution/pom.xml

```diff
@@ -63,4 +63,14 @@
             </build>
         </profile>
     </profiles>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.stratos</groupId>
+            <artifactId>org.apache.stratos.cep300.extension</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.stratos</groupId>
+            <artifactId>org.apache.stratos.cep.310.extension</artifactId>
```

Comment:
Why we have two diffrent naming conventions for artifactId? (org.apache.stratos.cep300.extension and org.apache.stratos.cep.310.extension) is that correct?